### DISCUSSION TITLE
Ideas for future SRS implementation

### DISCUSSION THREAD

**Discussion by hwgilbert16** [hwgilbert16/scholarsome#14] [Ideas]
I'll be starting work soon on our SRS implementation. 

Would love to hear ideas about how ours can be better than the existing options, such as Anki.

**Comment by mailtijmen** [comment]
Maybe adding flashcards through a csv? Now I can only start a new list using a csv or am I not doing this right?

**Comment by ssc2005** [comment]
a method to use fractions with a horizontal line would be great rather than having to rely on x/y

**Comment by nobility-zhang** [comment]
There is the concept of goals in any SRS system, but the goals of most systems are corresponding quantities, and few set the goal as learning time. I think time is a very important concept.Because the difficulty of these  flash cards is different, it is easy to complete today's goal for some simple ones, but for some difficult flash cards, I will not be able to complete the goal.In addition to judging from the number of times and accuracy, the time of learning is also a very important condition (Stay on the front of the flash card and remember the time)

There are many factors that determine the degree of forgetting in the RSA algorithm. I think recall time is very important.

I think the difference between reading time and recall time can be determined based on the number of words in the answer, which can prompt me to choose the degree of forgetting more correctly.
